components/console-backend-service/internal/domain/assetstore/file_service.go

```diff
@@ -0,0 +1,55 @@
+package assetstore
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kyma-project/kyma/components/asset-store-controller-manager/pkg/apis/assetstore/v1alpha2"
+)
+
+type fileService struct{}
+
+func (svc *fileService) FilterByExtensions(statusRef *v1alpha2.AssetStatusRef, filterExtensions []string) ([]*File, error) {
+	if statusRef == nil {
+		return nil, nil
+	}
+
+	if len(filterExtensions) == 0 {
```

Comment:
I'm wondering if the check shouldn't be done directly in the resolver. Because currently the method's name is inaccurate. Sometimes `FilterByExtensions` doesn't filter by extensions 😄